player/lua/osc.lua

```diff
@@ -383,7 +389,7 @@ local function ass_append_alpha(ass, alpha, modifier)
     end
 
     ass:append(string.format("{\\1a&H%X&\\2a&H%X&\\3a&H%X&\\4a&H%X&}",
-               ar[1], ar[2], ar[3], ar[4]))
+               math.floor(ar[1]), ar[2], ar[3], ar[4]))
```

Comment:
open question, should it be applied to all of them? And should it be floor or round would be better?

Reply:
I'm not familiar with this code but you can observe that only the first value has a decimal part, else the OSC would crash on Lua 5.4 without converting the following arguments. I don't think floor vs round matters over a 0-255 range, floor is just more convenient because there is no `math.round` (you have to do `math.floor(n + 0.5)`.

Reply:
`math.floor` can be called in `mult_alpha` to easily apply it to all 4.